portlet-tck_3.0/pom.xml

```diff
@@ -112,7 +119,10 @@
       
       <!-- Lines commented out below illustrate how the chromedriver web driver can be configured -->
       <test.browser>chrome</test.browser>
-      <test.browser.webDriver>c:/ntutil/chromedriver.exe</test.browser.webDriver>
+      <test.browser.webDriver>/usr/local/bin/chromedriver</test.browser.webDriver>
+		<!-- this binary below is only used for configuring your chromium install for headless use -->
+		<browser.binary>/Users/apple/Downloads/chrome-mac/Chromium.app/Contents/MacOS/Chromium</browser.binary>
+      <!-- test.browser.webDriver>c:/ntutil/chromedriver.exe</test.browser.webDriver -->

```

Comment:
@vsingleton, shouldn't all these changes be made via command-line options: `-Dtest.browser.webDriver=/usr/local/bin/chromedriver`. Also `browser.binary` should probably be null or empty by default and be prefixed with `test`: `<test.browser.binary></test.browser.binary>`